src/shogun/classifier/mkl/MKLMulticlass.cpp

```diff
@@ -34,9 +32,20 @@ MKLMulticlass::MKLMulticlass(float64_t C, std::shared_ptr<Kernel> k, std::shared
 	svm=NULL;
 	lpw=NULL;
 
+	init();
+}
+
+void MKLMulticlass::init()
+{
 	mkl_eps=0.01;
 	max_num_mkl_iters=999;
 	pnorm=1;
+
+	SG_ADD(&mkl_eps, "mkl_eps", "MKL Epsilon");
+	SG_ADD(&max_num_mkl_iters, "max_num_mkl_iters", "MKL max number of iterations");
+	SG_ADD(
+	    &pnorm, "mkl_norm", "MKL norm", ParameterProperties::NONE | ParameterProperties::CONSTRAIN,
+	    SG_CONSTRAINT(greater_than_or_equal(1.0)));
```

Comment:
@karlnapf added this to MKLMulticlass, should be fine no? Nothing was being exposed to parameter framework.

Reply:
yep, otherwise it is also not serialized and loading will reset it to the default setting